packer/builder_mock.hcl2spec.go

```diff
@@ -2,9 +2,10 @@
 package packer
 
 import (
+	"io"
```

Comment:
I believe we need to exclude the generated files from this 🤔

Reply:
I agree here; the code generator should generate well formatted code; but when it doesn't we should not make the tools fight :p automated tests won't ever pass otherwise.

Reply:
Ooh! Yeah I’ll make the adjustments on the exclusions.